lib/databases/postgresql

```diff
@@ -71,25 +71,42 @@ function configure_database_postgresql {
         pg_conf=/var/lib/pgsql/data/postgresql.conf
         # initdb is called when postgresql is first started
         sudo [ -e $pg_hba ] || start_service postgresql
+    elif is_clearlinux; then
+        # clearlinux code
+        pg_hba=/var/lib/pgsql/data/pg_hba.conf
+        pg_conf=/var/lib/pgsql/data/postgresql.conf
+        if ! sudo [ -e $pg_hba ]; then
+            start_service postgresql-install
+        fi
     else
         exit_distro_not_supported "postgresql configuration"
     fi
     # Listen on all addresses
     sudo sed -i "/listen_addresses/s/.*/listen_addresses = '*'/" $pg_conf
     # Set max_connections
     sudo sed -i "/max_connections/s/.*/max_connections = $MAX_DB_CONNECTIONS/" $pg_conf
+
+    if is_clearlinux; then
+        # Do ident auth for Unix domain socket connections, IPv4 clients and IPv6 clients.
+        sudo sed -i "s/reject/ident/" $pg_hba
```

Comment:
This is better now.
1. modify all "reject" to "ident", then the conf file looks normal.
2. use devstack old code to modify IPv4/IPv6 from "ident" to "password".

Reply:
This is better and clearer to read, thank you

Reply:
This is cleaner to read, thank you